node/tests/PubSub.test.ts

```diff
@@ -92,14 +93,17 @@ describe("PubSub", () => {
         pubsubSubscriptions2?:
             | GlideClientConfiguration.PubSubSubscriptions
             | GlideClusterClientConfiguration.PubSubSubscriptions,
+        decoder?: Decoder,
     ): Promise<[TGlideClient, TGlideClient]> {
         let client: TGlideClient | undefined;
 
         if (clusterMode) {
             try {
                 options.pubsubSubscriptions = pubsubSubscriptions;
+                options.defaultDecoder = decoder;
                 client = await GlideClusterClient.createClient(options);
```

Comment:
Instead of editing the input argument object.
```suggestion
                client = await GlideClusterClient.createClient({
                    pubsubSubscriptions = pubsubSubscriptions,
                    defaultDecoder = decoder,
                    ...options
                });
```

Reply:
I have tried your suggested implementation but the test failed.

Reply:
I have resolved the issues.